<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>888</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SmartCalc v.2.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/5Head.png</normaloff>:/icons/icons/5Head.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">

QMainWindow {
	font-size: 12px;
	background-color: #17181a;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">* {
	font-family: ubuntu;
	font-size: 45px;
	font-weight: 0px;
	color: white;
	text-align: center;
	vertical-align: center;
	border-radius: 22px;
	border: 0px;
}

QToolTip { 
	color: white;
	background-color: #17181a;
	font-size: 16px;
}</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>451</y>
      <width>321</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #303136;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_dot">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>340</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>451</y>
      <width>211</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #005db2;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QPushButton:disabled {
	background-color: #868686;
	color: #5a5a5a;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_mul">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_plus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_div">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_equals">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #1991ff;
	color: #b2daff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_min">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_mod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>191</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>mod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_x">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>521</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 25px;
	border-radius: 28px;
	background-color: #303136;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QPushButton:disabled {
	background-color: #868686;
	color: #5a5a5a;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_cos">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>cos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_acos">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>acos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>sin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_asin">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>asin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ln">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>ln</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_tg">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>tg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_atg">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>atg</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_log">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="calcButton">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Basic</string>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="graphicButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Extended</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="creditButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>81</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Credit</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="depositButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Deposit</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QRadioButton:indicator {
	background-color: #17181a;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</normaloff>../../../Documents/SmartCalc_UI/resources/calc_off.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_input_x">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>switch input</string>
     </property>
    </widget>
    <zorder>pushButton_cos</zorder>
    <zorder>pushButton_acos</zorder>
    <zorder>pushButton_sin</zorder>
    <zorder>pushButton_asin</zorder>
    <zorder>pushButton_ln</zorder>
    <zorder>pushButton_tg</zorder>
    <zorder>pushButton_atg</zorder>
    <zorder>pushButton_log</zorder>
    <zorder>calcButton</zorder>
    <zorder>graphicButton</zorder>
    <zorder>depositButton</zorder>
    <zorder>creditButton</zorder>
    <zorder>pushButton_input_x</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>349</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #005db2;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}

QPushButton:disabled {
	background-color: #868686;
	color: #5a5a5a;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_pow">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_brackets">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>( )</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sqrt">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>√</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #B0C4DE;
	color: #005db2;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_C">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_del">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>0</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/clear.png</normaloff>icons/clear.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Input">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>20</y>
      <width>511</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>511</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>991</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>ubuntu</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>110</y>
      <width>451</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>570</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	font-size: 25px;
}

QSpinBox {
	border-radius: 15px;
	background-color: #303136;
	color: #24a5ff;
}

QSpinBox#spin::up-arrow  {
min-width: 19px;
min-height: 14px;
max-width: 19px;
max-height: 14px;
height: 19px;
width: 14px;
}

QPushButton {
	border-radius: 28px;
	background-color: #303136;
	color: #24a5ff;
}

QPushButton:pressed {
	background-color: #004787;
	color: #1a7cc0;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="yMax">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="yMin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>-10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/icons/icons/graphic.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="xMax">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="xMin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>111</width>
       <height>61</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>-1000000</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
     <property name="value">
      <number>-10</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>ubuntu</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 45px;
}</string>
     </property>
     <property name="text">
      <string>X =</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Input_x">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>580</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ubuntu</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	color: #828282;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="error_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	color: red;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
